src/md/src/GettingStarted/Integrations.md

```diff
@@ -11,9 +11,9 @@ Users can reinforce ReportPortal with adding additional integrtions with:
 * [Sauce Labs](https://reportportal.io/docs/Sauce-Labs-integration)
 * [E-mail server](https://reportportal.io/docs/E-mail-server-integration)
 * [LDAP](https://reportportal.io/docs/LDAP-Auth-integration)
-* [Active Directory](https://reportportal.io/docs/Active-Directory-Auth-integration)
```

Comment:
Maybe better add `-integration` postfix in the `Plugins.md` file to be more consistent